cluster/gce/util.sh

```diff
@@ -644,10 +644,24 @@ function kube-down {
 
   echo "Bringing down cluster"
 
-  gcloud preview managed-instance-groups --zone "${ZONE}" delete \
+  deleteCmdOutput=$(gcloud preview managed-instance-groups --zone "${ZONE}" delete \
     --project "${PROJECT}" \
     --quiet \
-    "${NODE_INSTANCE_PREFIX}-group" || true
+    "${NODE_INSTANCE_PREFIX}-group" || true)
+  if [[ "$deleteCmdOutput" != ""  ]]; then
```

Comment:
Yes, both statements skip over the deletion check, which is intended--if the group ID doesn't exist, for example, it doesn't make sense to wait for deletion to complete.